### ISSUE TITLE
Weapon stored Backpack exploit

### ISSUE THREAD

**Issue by Looking4bros** [tgstation/TerraGov-Marine-Corps#14884]
### Description

Fucking roomba ate my flamethrower backpack, Got it from the storage it dumps everything into, and now the backpack will draw the flamer from anywhere on the map while not appearing inside the backpack itself. Full storage space and never loose your flamer.

Tested with Rocketbag as well, works with Recoilless rifle rocket backpack.
### Test Merges

Server-Side Code Modifications: https://tgstation13.org/parsed-logs/sybil/server_side_modifications.dm

Round ID: [25336](https://scrubby.melonmesa.com/round/25336)
BYOND Version: 515.1626
Server revision compiled on: 2024-01-21T20:33:45+00:00
Master commit: [266cdf5f49f15f22c46d6154b8e0dd73dc8f50cc](https://github.com/tgstation/TerraGov-Marine-Corps//commit/266cdf5f49f15f22c46d6154b8e0dd73dc8f50cc)
The following pull requests are currently test merged:
[#14850: 'chat style changes' by Neocloudy at commit 91fce7d484](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14850)
[#14880: 'Fixes certain click interactions' by Xander3359 at commit d89106da81](https://github.com/tgstation/TerraGov-Marine-Corps//pull/14880)

Local commit: 518fe027815308d9d111ed5a0167f261379a7169
Server tools version: 6.1.2

### Reproduction Steps

1. Get flamer backpack (or any bag that is intended to draw a specific weapon from it storage)
2. Roomba eats it
3. Get Bag from storage
4. See bag sprite still has flamer(or RR) despite no flamer inside, still quickdraw flamer from flamerless backpack.

### Screenshots

Can put video in Discord if needed.

**Comment by Oscari-pog** [comment]
Absolutely fucking flabbergasted at how this even occurs

**Comment by Tailiik** [comment]
Following up on this, I was in a round where this happened and I was goofing around with it with the player who did it. This bug somehow creates two instances of the flamer that are quantum entangled. Player A drops the flamer, player B picks it up. Player B's flamer is now real while player A's flamer is fake, flamer B can fire and works as expected, until....player A draws the flamer again, which makes player B's flamer fake and unable to fire, but it still exists in their hand. Player B ends up with some funky mob changes that persist after they drop the fake flamer, like your walking speed having the wielded flamer slowdown while not holding anything, but it is fixed by wielding a REAL gun that isn't quantum entangled. 

The weird quantum entangled item fuckery can happen in other circumstances too, like once another player and I picked up a gun at the same exact time and we gave it the weird "dupe" effect. (See video below) When I dropped my "fake" gun in the video the "real" gun underneath the other guy disappeared, and since the gun doesn't have a bugged quickdraw bag (like rr or flamer) to pull out of the quantum state ended.

It also can happen with webbings, if you have a webbing on your uniform already and you attach a new one with nothing in your other hand, you end up with an invisible "fake" webbing and a visible "real" webbing. The "real" one can be picked up and opened, but the invisible "fake" one being dropped causes them to resolve into just one item and the state ends. I noticed this happening with webbings a long time ago but now I've noticed this as a pattern that can happen multiple ways.

[Quantum entangled caliver](https://i.imgur.com/HVNkd6a.mp4)
